.github/workflows/test.yml

```diff
@@ -8,11 +8,15 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest]
-        ruby: [2.7]
+        ruby:
+          - '2.7'
+          - '3.0'
+          - '3.1'
+          - '3.2'
```

Comment:
This will run tests 4 * 2 times, that's quite a lot.
In my opinion the most important thing is to check whether the code works on a few Ruby versions, the OS shouldn't matter that much.

I suggest that you remove `os` from the matrix and run tests only on `ubuntu-latest`.

Reply:
Agreed. Ruby shouldn't really differ between operating systems, so whether you run them on OSX or on Linux it's the same.
Besides, I've seen that in Aliquot tests are run only for `ubuntu-latest`.